guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,12 +4,49 @@
+
+*Host bulk actions* - You can now select one or more hosts and click the vertical ellipsis menu to perform the following actions:
+Manage content
+Packages (install / remove / upgrade)
+Errata
+Content source
+Content view environments - change the content view and lifecycle environment of hosts
+
+*Single-host actions* - In addition to actions added by Foreman, you can click the vertical ellipsis in the rightmost table column (without selecting the host) to change content view environments on a single host.
+
+*Column selector* - You can now click Manage Columns and customize columns displayed in the table. The column options are the same as those on the legacy All Hosts page. Any changes made here will also reflect on the legacy All Hosts page, and vice versa.
+
+=== Multiple Content Views ===
+
+Assigning multiple content view environments to hosts
+
+You can now assign multiple content view environments to a host via subscription-manager or Hammer. A multi-environment host will have access to enabled repositories from all of its assigned content view environments. This strategy may be more convenient than composite content views for instances where frequent publishes are required.
+
+Via hammer, pass the `content-view-environments` param to `hammer host update` in a comma-separated list in the format `<lifecycle_environment_label>/<content_view_label>` . Ex `hammer host update --id xxx --content-view-environments lce1/cv1,lce2/cv2`
+
+On the host, run `subscription-manager environments --set` and pass in a comma-separated list. Ex. `subscription-manager environments --set lce1/cv1,lce2/cv2` .
+To see the list of possible content view environments, use subscription-manager environments --list
+To see the currently assigned content view environment(s), use subscription-manager environments --list-enabled
+
+The web UI does not yet support assigning multiple environments. However, multi-environment hosts will now be displayed in the web UI on the new All Hosts page and new host details page.
+
+On the new All Hosts page, there is a new 'Content view environments' column available to add. This will display all assigned environments for a host. The existing 'Content view' and 'Lifecycle environment' columns will continue to work, but will only display the first assigned environment.
+
+If you reassign a host's content view environments using the web UI, the host will no longer be a multi-environment host.
+
+Assigning or registering a host to multiple environments requires the Allow multiple content views Setting to be on. If you turn this setting off, existing multi-environment hosts will continue to work, but you will not be able to assign multiple environments to hosts going forward.
+
+You can also assign multiple environments at registration using subscription-manager register --environments, but global registration and activation keys are not yet supported.
 
 [id="katello-upgrade-warnings"]
 == Upgrade Warnings
```

Comment:
https://github.com/theforeman/foreman-documentation/pull/3180 should also be included. If @ehelms doesn't finish it then you should take over the effort.

Reply:
@ehelms it looks like they want you to rebase on the 3.12 branch. https://github.com/theforeman/foreman-documentation/pull/3180#pullrequestreview-2254314673

Did you want me to just take your note and add this to mine?